Rss to html

Lauren-10 opened the pull request:
Step 1 of the pipeline, looking to incorporate a feature to read xml files in addition to rss feeds.

Lauren-10 committed:
Task 1 completed, files attached below from test
run

Lauren-10 committed:
deleted a data file

Lauren-10 committed:
rssToHtml script

Lauren-10 committed:
step 1 of pipeline

c-ret2 commented:
Looks great! the column names that we need for SQL compatibility are "publication, url, article_title, date"

Lauren-10 committed:
fixed column headers

Lauren-10 committed:
Commented out some additional code that increases
functionality and gives the option for different
inputs and outputs (either df or csv)

Lauren-10 committed:
Moved to source directory
